resources/collections.meta-storm.xml

```diff
@@ -2,21 +2,21 @@
 <meta-storm xmlns="meta-storm">
     <collections>
         <attributeArgument
-            name="entity-role"
-            className="\Cycle\Annotated\Annotation\Entity"
-            argumentIndex="0"
+            name="cycle/annotated:entity-role"
+            class="\Cycle\Annotated\Annotation\Entity"
+            argument="0"
         />
         <attributeClass
-            name="entity-class"
-            className="\Cycle\Annotated\Annotation\Entity"
+            name="cycle/annotated:entity-class"
+            class="\Cycle\Annotated\Annotation\Entity"
         />
         <attributeClass
-            name="entity-embeddable-class"
-            className="\Cycle\Annotated\Annotation\Embeddable"
+            name="cycle/annotated:entity-embeddable-class"
+            class="\Cycle\Annotated\Annotation\Embeddable"
         />
         <attributeClass
```

Comment:
Seems it's unused

Reply:
It's used there:
https://github.com/cycle/annotated/blob/67917849e4f23710917d21a47a6177514e11d9bb/resources/relations.meta-storm.xml#L120-L125

Reply:
Yes, it looks like a extra copy-paste